index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@800&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+    <title>Document</title>
+</head>
+<body>
+    <div class="flex-container header">
+        <div class="flex-item">
+            <a href="#">Portfolio</a>
+        </div>
+        <div class="flex-item">
+            <a href="#">About</a>
+        </div>
+        <div class="flex-item">
+            <a href="#">Contact</a>
+        </div>
+    </div>
+    <div class="flex-container main">
+        <div class="flex-box">
+            <h1 class="intro">Hey There. I'm Ahmad.</h1>
+            <h1 class="profession">I'm a Full-stack Web Developer.</h1>
+            <p>I can help you build interactive and responsive websites. To be sure about the quality of my work, you may view my portfolio.</p>
+        </div>
+        <div>
+            <div class="flex-container">
```

Comment:
Kindly implement the button icon in the Figma design with all it's respective styling, to enable a pixel-perfect design 🙂.

- Your design 
<img width="167" alt="wrongmain" src="https://user-images.githubusercontent.com/41163571/219195228-89669ea3-36ee-4ce5-85a4-1ebddbbc32d0.png">

- Figma design
<img width="199" alt="rightarrow" src="https://user-images.githubusercontent.com/41163571/219195516-61fb6f35-cf23-485e-b400-e9e59033dcee.png">